backend/cmd/api/main.go

```diff
@@ -135,7 +136,8 @@ func getLoadersForProfile(queries *sqlc.Queries, profileID uuid.UUID) *common.Pr
 				Column2:   ids,
 			})
 		}, batchloaders.WithMemoryCache(time.Second*5)),
-		AchievementAchievedAtLoader: batchloaders.New(profileQueries.GetAchievementsAchievedAt, batchloaders.WithMemoryCache(time.Second*5)),
+		AchievementAchievedAtLoader:   batchloaders.New(profileQueries.GetAchievementsAchievedAt, batchloaders.WithMemoryCache(time.Second*5)),
+		GetSelectedAlternativesLoader: batchloaders.New(profileQueries.GetSelectedAlternatives, batchloaders.WithMemoryCache(time.Second*10)),
```

Comment:
Well, 10 seconds is a bit much if I'm supposed to be stateless here.

Reply:
But I'm considering to only load it once for a better UX on slower networks. But technically it would be best to reduce this down to the minimum.

Reply:
I can see if I can easily evict that key from cache when you submit an answer.

Reply:
I mean 10s is basically since you loaded the answer so you would have to load answer select, submit, go back. Not impossible but somewhat reasonable I feel, I'll reduce it to 3 seconds for the PR, and then look into better options after. OK?

Reply:
I was thinking more if I select something, go to the next task and then back to edit.. But it's a better UX to just remember the state so that's what I'm going to do.

Reply:
It's easy to clear cache, doing that on submission now.